packages/sw-background-sync-queue/src/lib/broadcast-manager.js

```diff
@@ -0,0 +1,21 @@
+import {defaultBroadcastChannelName} from './constants';
+let bcmanager;
+
+function initiazileBroadcastManager(channelName) {
+	bcmanager = new BroadcastChannel(channelName || defaultBroadcastChannelName);
+}
+
+function broadcastMessage({type, id, url}) {
+	bcmanager && bcmanager.postMessage({
```

Comment:
Consider using the [`assert` library](https://github.com/GoogleChrome/sw-helpers/blob/master/lib/assert.js) to validate these parameters if they're required.